index.js

```diff
@@ -1,12 +1,16 @@
 var kimi = require( 'kimi' ),
     f1Parser = require( 'f1-parser' ),
     getTween = require( 'tween-function' ),
-    noop = require( 'no-op' );
+    noop = require( 'no-op' ),
+    _ = require('lodash');
```

Comment:
I don't have an issue with the functionality of `lodash` but if you take the context of `f1` bringing in `lodash` maybe an issue. I know it's a dep right now in other parts but I want to get rid of it because it uses so many kb.